integration-tests/pages/Projects.ts

```diff
@@ -0,0 +1,369 @@
+          By.className('ant-menu ant-menu-light ant-menu-root ant-menu-horizontal'),
+        );
+        const tabs = await tabList.findElements(By.className('ant-menu-item'));
+        tab === 'Access rights' && (await tabs[1].click());
+        const body = modal.findElement(By.className('ant-modal-body'));
+
+        resolve(body);
+      } catch (error) {
+        reject(`Error on Project.openProjectSettings('${name}'): ${error}`);
+      }
+    });
+  }
+
+  addUserToProject(projectName: string, userName: string, userRole: UserRoleType) {
+    return new Promise<void>(async (resolve, reject) => {
+      try {
+        // open settings
+        const body = await this.openProjectSettings(projectName, 'Access rights');
+        const selections = await body.findElements(By.className('ant-select-selection__rendered'));
+
+        // add the user
+        let userFound = false;
+        await selections[0].click();
+        await this.sleep(100);
+        const userDropdown = await this.driver.findElement(
+          By.className(
+            'ant-select-dropdown ant-select-dropdown--single ant-select-dropdown-placement-bottomLeft',
+          ),
+        );
+        const users = await userDropdown.findElements(By.css('li'));
+        for (const user of users) {
+          const text = await user.getText();
+          if (userName === text) {
+            await user.click();
+            userFound = true;
+          }
+        }
+        !userFound &&
+          reject(
+            `Error on Project.addUserToProject('${projectName}', '${userName}', '${userRole}'): User not found`,
+          );
+
+        // add the role
+        let roleFound = false;
+        await selections[1].click();
+        await this.sleep(100);
+        const bothDropdowns = await this.driver.findElements(
+          By.className(
+            'ant-select-dropdown ant-select-dropdown--single ant-select-dropdown-placement-bottomLeft',
+          ),
+        );
+        const roleDropdown = bothDropdowns[1];
+        const roles = await roleDropdown.findElements(By.css('li'));
+        for (const role of roles) {
+          const text = await role.getText();
+          if (userRole === text) {
+            await role.click();
+            roleFound = true;
+          }
+        }
+        !roleFound &&
+          reject(
+            `Error on Project.addUserToProject('${projectName}', '${userName}', '${userRole}'): Role not found`,
+          );
+
+        // click the add button
+        const addButton = await body.findElement(By.className('ant-btn ant-btn-primary'));
+        await addButton.click();
+        await this.alerts('success', userName + 'was added');
+        resolve();
+      } catch (error) {
+        reject(
+          `Error on Project.addUserToProject('${projectName}', '${userName}', '${userRole}'): ${error}`,
+        );
+      }
+    });
+  }
+
+  removeUserToProject(projectName: string, userName: string) {
```

Comment:
this should be removeUser*From*Project

Reply:
Just change it and commit again. Should be automatically included in merge request